docs/reference/sql/alter.md

```diff
@@ -118,6 +122,30 @@ ALTER TABLE monitor MODIFY COLUMN load_15 STRING;
 
 The modified column cannot be a tag (primary key) or time index, and it must be nullable to ensure that the data can be safely converted (returns `NULL` on cast failures).
 
+### Set column default value
+
+Set a default value for an existing column:
+
+```sql
+ALTER TABLE monitor MODIFY COLUMN load_15 SET DEFAULT 0.0;
+```
+
+Set a string default value:
+
+```sql
+ALTER TABLE monitor MODIFY COLUMN `status` SET DEFAULT 'active';
+```
+
+The default value will be used for new rows when no explicit value is provided for the column during insertion.
+
+Remove the default value from a column:
+
+```sql
+ALTER TABLE monitor MODIFY COLUMN load_15 DROP DEFAULT;
+```
+
+After dropping the default value, new rows will require an explicit value for the column or will use `NULL` if the column allows it.
```

Comment:
After drop default the column will use null as default. We only allow dropping default for a nullable column.

Reply:
[08a3b5b](https://github.com/GreptimeTeam/docs/pull/2010/commits/08a3b5b4808d8f1ebde82cdf2c6f2a7016d25911)